python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -750,6 +789,13 @@ module ClassValue {
     ClassValue complex() {
         result = TBuiltinClassObject(Builtin::special("complex"))
     }
+    
+    /** The builtin class '(x)range' */
+    ClassValue rangeType() {
+        result = TBuiltinClassObject(Builtin::special("xrange"))
+        or
+        major_version() = 3 and result = TBuiltinClassObject(Builtin::special("range"))
+    }
```

Comment:
I didn't spot this before, but we actually have `ClassValue::range()` that does exactly this :blush:

Reply:
Ya, I think I added this type to this PR before I did the foresight modernizations, so I'll remove this today and use the foresight one.

Reply:
Ok, done.